src/plugins/auto/auto_schedule.cpp

```diff
@@ -391,7 +391,7 @@ void AutoSchedule::TryToLoadNetWork(AutoLoadContext& context, const std::string&
         std::lock_guard<std::mutex> lock(_autoSContext->_confMutex);
         context.deviceInfo = _autoSContext->_plugin->SelectDevice(deviceList,
                 context.networkPrecision, _autoSContext->_modelPriority);
-    } catch (const std::exception& e) {
+    } catch (...) {
```

Comment:
why? catching `...` is bad practice

Reply:
Do we need add some code to access `e` to avoid unused variable warning?
BTW, there are several similar code in current openvino source code: https://github.com/openvinotoolkit/openvino/blob/41866ea51ac84bee88e0f489ba553ca5354b0d98/src/core/src/node.cpp#L100

Reply:
such places also need to be fixed

Reply:
@panhaiqi  please help submit a new PR to resolve all `catch(...)` issue.